docs/resources/mws_network_connectivity_config.md

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Deployment"
+---
+# databricks_mws_network_connectivity_config Resource
+
+-> **Note** Initialize provider with `alias = "mws"`, `host = "https://accounts.azuredatabricks.net"` and use `provider = databricks.mws` for all `databricks_mws_*` resources.
+
+-> **Public Preview** This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html) in Azure. This feature is currently not available in other clouds.
```

Comment:
Feature is currently in private preview. Can we hold this change until the release in early October?

Reply:
Just mark this as a Draft PR?

Reply:
Hi @grayluck, thanks for the PR, given it is going to be on hold, I am marking this as draft. Please convert back to ready for review once that is done.

Reply:
@tanmay-db terraform support is public preview release blocker. what is the proper process here?

Reply:
Proper process in that case is the support to be added in terraform before release. Just to confirm, what do you mean by?

> Can we hold this change until the release in early October?

You mean to have this change as part of terraform release just before public preview announcement? 

Also when is public preview happening?

Reply:
Currently targetting mid October. I think we can merge the change around the time window.

Reply:
Resuming the review of this PR - we are target release end of October.